tests/suites/test_suite_ecp.function

```diff
@@ -1433,3 +1433,69 @@ exit:
     mbedtls_free(p);
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS */
+void ecp_mod_mul_inv(char *input_A, int id, int ctype, int iret)
+{
+    int ret;
+    mbedtls_mpi_mod_modulus m;
+    mbedtls_mpi_mod_modulus_init(&m);
+    mbedtls_mpi_mod_residue A = { NULL, 0 };
+    mbedtls_mpi_mod_residue A_inverse = { NULL, 0 };
```

Comment:
I would suggest we keep the residue naming notation consistent.

    mbedtls_mpi_mod_residue rA = { NULL, 0 }; // For input
    mbedtls_mpi_mod_residue rX = { NULL, 0 }; // For result
    mbedtls_mpi_mod_residue rA_inv= { NULL, 0 }; // For inverse input